README.md

```diff
@@ -120,7 +112,7 @@ services:
         - MARKLOGIC_ADMIN_PASSWORD_FILE=mldb_admin_password
         - TZ=Europe/Prague
       volumes:
-        - ~/data/MarkLogic:/var/opt/MarkLogic
+        - MarkLogic:/var/opt/MarkLogic
```

Comment:
Do you think our naming convention for these yaml files should include a prefix or suffix containing what type of cluster it is? 

I can forsee clients wanted to run through the whole readme and having a few Volumes on their machine from the single node and three node example. 

Maybe something like 
`Marklogic_sn:/var/opt/MarkLogic`
for here and 
`MarkLogic1_mn:/var/opt/MarkLogic`  for below in the 3-node example. 

No big deal either way just a thought.

Reply:
If we do end up going with it, we would also want to change the docker compose example.

Reply:
Agreed. I wanted to bring this up as well after we merged into release branch.